cudapoa/src/application_parameters.cpp

```diff
@@ -109,6 +111,17 @@ ApplicationParameters::ApplicationParameters(int argc, char* argv[])
         throw std::runtime_error("band-width must be positive");
     }
 
+    if (!banded && adaptive)
+    {
+        throw std::runtime_error("adaptive alignment cannot run with full alignment");
+    }
+
+    if (adaptive && default_bandwidth)
+    {
+        // Bump up the default bandwidth as it is 256 for static banded which is too low
+        band_width = 2048;
```

Comment:
I added this bump according to our previous discussion but it is not needed for the sample-xyz.txt files.

Reply:
it helps to clarify the role of `band_width` in adaptive-banded alignment. A note to indicate this parameter is used to determine maximum score matrix size. I think 2048 might still be conservative, but we can have a better understanding of this once we check the results from benchmark tests.